src/utils/filter.js

```diff
@@ -0,0 +1,14 @@
+import {FilterType} from '../constants.js';
+import {
+  isEventPointInFuture,
+  isEventPointInPast
+} from './event-point.js';
+
+
+const filter = {
+  [FilterType.EVERYTHING] : (eventPoints) => eventPoints.filter((point) => point),
+  [FilterType.FUTURE] : (eventPoints) => eventPoints.filter((point) => isEventPointInFuture(point)),
```

Comment:
Здесь можно немного короче записать передав коллбек
```suggestion
  [FilterType.FUTURE] : (eventPoints) => eventPoints.filter(isEventPointInFuture),
```